modules/runner/src/integrationTest/java/org/apache/ignite/internal/benchmark/AbstractMultiNodeBenchmark.java

```diff
@@ -90,25 +90,35 @@ public void nodeSetUp() throws Exception {
         startCluster();
 
         try {
-            var queryEngine = igniteImpl.queryEngine();
+            // Create a default zone on the cluster's start-up.
+            createDefaultZoneOnStartup();
```

Comment:
It seems there is a clash in terminology. There is a default zone that is created by the cluster on initialization automatically (implicitly). Here, another zone is created explicitly, so it probably should not be named a 'default zone'. How about just `createZoneOnStartup()`?

Reply:
Ok, it makes sense. Changed to `createDistributionZoneOnStartup`. Thanks for pointing this out!